### PR TITLE
Fix tests and refactor colors case notation

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
## Description
Fixed tests, did a little refactoring.

image

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/composables/useColors.ts [hunk 1]
```diff
@@ -88,6 +88,10 @@ export const useColors = () => {
       return prop
     }
 
+    if (isCSSVariable(prop)) {
```
Comment (aluarius) [inline packages/ui/src/composables/useColors.ts:91]: Why not:

image

Or it wa changed before?

**Review by aluarius — APPROVED** [review]

File: packages/ui/src/composables/useColors.ts (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/composables/useColors.ts:91]: It's wrong test condition, not composables. 

was: [['var(--va-primary)', undefined, false], '#2C82E0'],
now: [['var(--va-info)', undefined, false], 'var(--va-info)'],

It shouldn't return variable but hex.

File: packages/docs/src/config/theme-config.ts [hunk 2]
```diff
@@ -14,29 +14,29 @@ export const COLOR_THEMES: Record<ThemeName, Record<string, string>> = {
...
     primary: '#2550C0',
     secondary: '#767C88',
-    success: '#40e583',
-    info: '#2c82e0',
+    success: '#40E583',
+    info: '#2C82E0',
     danger: '#DE1041',
     warning: '#FFAC0A',
 
-    backgroundPrimary: '#ffffff',
```
Comment (asvae) [inline packages/docs/src/config/theme-config.ts:22]: Can't we use lowercase colors?

In all other places it's lowercase

Comment (m0ksem) [reply packages/docs/src/config/theme-config.ts:22]:
```ts
{
 danger: '#f0f',
 'background-primary': '#fff
}
```

It doesn't look too nice. BTW, it is not case-sensitive.

**Review by aluarius — APPROVED** [review]

File: packages/docs/src/config/theme-config.ts (hunk 2 above)
Comment (asvae) [reply packages/docs/src/config/theme-config.ts:22]: @m0ksem meant hexes 😄, so `#ffac0a` instead of `#FFAC0A`.

**Pull request merged**